microsoft/PowerToys
# [Workspaces] Sequential launch

## Summary of the Pull Request

Launching the next instance of the app only after the previous one has appeared. This ensures that we avoid errors when launching other instances (e.g. "The app is already running") and placing app instances in the correct positions.


## PR Checklist

- [x] **Closes:** https://github.com/microsoft/PowerToys/issues/34675, https://github.com/microsoft/PowerToys/issues/34730, https://github.com/microsoft/PowerToys/issues/35010


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed

* Launch a workspace with several instances of the app, e.g., Outlook. Verify all instances are launched without error, and placed correctly.
* Verify canceling the launch works as expected.
